pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -77,15 +89,29 @@ func (enqueue *enqueueAction) Execute(ssn *framework.Session) {
 	emptyRes := api.EmptyResource()
 	nodesIdleRes := api.EmptyResource()
 	for _, node := range ssn.Nodes {
-		nodesIdleRes.Add(node.Allocatable.Clone().Multi(1.2).Sub(node.Used))
+		nodesIdleRes.Add(node.Allocatable.Clone().Multi(1.2))
+		nodeUsedRes.Add(node.Used)
+	}
+
+	//deltaRes will equal to the bigger resource consumption
+	deltaRes := api.EmptyResource()
+	if pgUsedRes.Less(nodeUsedRes) {
+		deltaRes = nodeUsedRes
+	} else {
+		deltaRes = pgUsedRes
```

Comment:
I don-t think this's correct: `node.Used` resources also include the Pod out of Volcano control.

Reply:
This is the logic before my changes:
```
	for _, node := range ssn.Nodes {
		nodesIdleRes.Add(node.Allocatable.Clone().Multi(1.2).Sub(node.Used))
	}

```
what's your suggestion here?